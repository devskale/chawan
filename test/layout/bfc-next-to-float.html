<!DOCTYPE html>
<div style="float: left">
first
</div>
<div style="display: flow-root">
<div style="width: 100%; background: red">second</div> second second second second second second second second second second second second second second second
</div>
<div style="display: flow-root">
third
</div>
<div style="float: left">
fourth
</div>
<div style="display: flow-root">
fifth
</div>
