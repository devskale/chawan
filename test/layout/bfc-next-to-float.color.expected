first[48;2;255;0;0msecond                                                                     [49m
     second second second second second second second second second second
     second second second second second
third
fourthfifth
